docker/worker/Pipfile

```diff
@@ -11,7 +11,8 @@ PyYAML = "*"
 google-cloud-storage = "*"
 redis = "*"
 osv = {editable = true, path = "./../.."}
-pygit2 = ">=1.14.0"
+pygit2 = "==1.14.1"
```

Comment:
Pinning is needed here as 1.15.0 causes breaking changes. This will be resolved in a followup PR.